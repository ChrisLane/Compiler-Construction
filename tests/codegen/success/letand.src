main() {
    let x = (true && true) {
        x;
    };
}
