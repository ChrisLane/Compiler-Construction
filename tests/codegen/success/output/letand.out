ld 1
st r1
ld 1
st r2
and r1, r2
st r1
mv r1, r2
mv r2, r1
ld r1
